dev/storage_base.h

```diff
@@ -450,6 +451,18 @@ namespace sqlite_orm {
                 return this->connection->retain_count() > 0;
             }
 
+            /*
+             * checks whether there is a transaction in place
+             */
+            bool in_transaction() const {
```

Comment:
better call this function `get_autocommit` to make it recognizable by people who know `sqlite3_get_autocommit` API. This is how all other API designed in `storage_t`. Also use `get_connection` instead to obtain a connection for sure

Reply:
no problem with this!

Reply:
please rename it to `get_autocommit` or remove it if it is not used

Reply:
Done!